src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -5,46 +5,50 @@ id: backup
 sidebar_label: Backup and Restore
 ---
 
-Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
-This guide outlines the steps to recover an Okteto instance in such scenarios.
+This guide outlines the steps to recover an Okteto instance in the event of an unexpected failure and for disaster recovery preparedness.
 
-## Critical Resources
+## Okteto Metadata
 
-To effectively recover an Okteto instance, focus on the following critical resources:
+The most important resources to backup so you can effectively recover an Okteto instance are:
 
-- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
-- **Secrets**: Recreate all secrets within the `okteto` namespace.
-- **Custom Resources**: Restore Custom Resources (CRs) corresponding to Okteto's CRDs. They can be identified with the label `app.kubernetes.io/part-of: okteto`.
+- **Service Accounts**: Okteto creates a Kubernetes service account for every user that logins into Okteto. Backup all the service accounts in the `okteto` namespace with the label `dev.okteto.com: "true"`.
+- **Secrets**: backup all the secrets in the `okteto` namespace. Okteto uses secrets to store Okteto Secrets, tokens, certificates, ssh keypairs, ...
+- **Custom Resources**: backup Custom Resources in the `okteto` namespace corresponding to the Okteto's CRDs. Okteto CRDs can be identified with the label `app.kubernetes.io/part-of: okteto`. Okteto uses CRDs to store your Catalog, Container Registry Credentials, ...
+
+We recommend to backup these resources every 24h.
 
 ## Recovery Process
 
 You can follow two different recovery processes:
 
-- **Light recovery processes**: recovers all the users’ metadata, tokens and secrets, but it doesn’t recover users’ namespaces and volumes. All this information is available on the `okteto` namespace.
-- **Full recovery processes**: recovers not only the `okteto` namespace, but also all the users’ namespaces and preview environments, including data in persistent volumes.
+- **Okteto Metadata recovery processes**: recovers all the users’ metadata, tokens, secrets and custom resources. It doesn’t recover users’ namespaces, preview environments and volumes.
+- **Okteto Metadata and Users' Namespaces recovery processes**: recovers not only the Okteto metadata, but also all the users’ namespaces, preview environments, and volumes.
```

Comment:
```suggestion
- **Okteto Metadata and Users' Namespaces recovery process**: recovers the Okteto metadata, namespaces, preview environments, and volumes for all users. This is a full and complete recovery of the Okteto instance
```

Some suggestions to clarify but making some assertions I'm not sure of. Is this in fact a full recovery?